client/src/app/app.component.ts

```diff
@@ -1,46 +1,37 @@
 import { Component, OnInit, HostBinding } from '@angular/core';
 import { Router, NavigationStart, NavigationEnd, NavigationError, NavigationCancel } from '@angular/router';
-import { SlimLoadingBarService } from 'ng2-slim-loading-bar';
+import { LoadingBarService } from './shared/loading-bar/loading-bar.service';
 
 declare const require: any;
 const { version } = require('../../package.json');
 
 @Component({
   selector: 'ml-app',
   template: `
-    <ng2-slim-loading-bar color="#FFC107" height="3px"></ng2-slim-loading-bar>
+    <ml-loading-bar></ml-loading-bar>
     <router-outlet></router-outlet>
   `,
   styles: [
     `
     :host {
       display: block;
     }
-
-    ng2-slim-loading-bar {
-      height: 3px;
-      position: absolute;
-      top: 0;
-      left: 0;
-      right: 0;
-      z-index: 6;
-    }
```

Comment:
I think we still need those styles for `ml-loading-bar` so that the UI doesn't "jump" (which is does right now)